app/src/main/java/org/oppia/android/app/spotlight/SpotlightTarget.kt

```diff
@@ -0,0 +1,72 @@
+package org.oppia.android.app.spotlight
+
+import android.view.View
+import org.oppia.android.app.model.Spotlight.FeatureCase
+
+/**
+ * Data class to hold a [SpotlightTarget].
+ *
+ * @param anchor the view that should be spotlit
+ * @param hint the hint text that should be shown on the spotlight overlay
+ * @param shape [SpotlightShape] of the spotlight
+ * @param feature The [Spotlight] feature-case corresponding to the spotlight target
+ */
+data class SpotlightTarget(
+  val anchor: View,
+  val hint: String = "",
+  val shape: SpotlightShape = SpotlightShape.RoundedRectangle,
```

Comment:
Why default these? It seems like they're always being defined in SpotlightFragment, anyway (prefer minimizing defaulting since that's technically complicating your class's API contract).